crates/bevy_scene/src/bundle.rs

```diff
@@ -0,0 +1,66 @@
+use bevy_asset::Handle;
+use bevy_ecs::{
+    bundle::Bundle,
+    change_detection::ResMut,
+    entity::Entity,
+    prelude::{Changed, Component},
+    system::Query,
+};
+use bevy_transform::components::{GlobalTransform, Transform};
+
+use crate::{DynamicScene, InstanceId, Scene, SceneSpawner};
+
+#[derive(Default, Component)]
+pub struct HasSceneInstance(Option<InstanceId>);
+
+#[derive(Default, Bundle)]
+pub struct SceneBundle {
+    pub scene: Handle<Scene>,
+    pub instance_id: HasSceneInstance,
+    pub transform: Transform,
```

Comment:
Should this use the TransformBundle instead?

Reply:
No: we should try to avoid nested bundles where feasible.